core/urls.py

```diff
@@ -31,4 +33,6 @@
     path('friendrequest/', friend_request, name='friendrequest'),
     path('login/', login, name='login'),
     path('unfollow/', handle_unfollow_request, name='unfollow'),
+    path('index_bundle.js', get_indexjs),
+    re_path(r'^', TemplateView.as_view(template_name='index.html')),
```

Comment:
Can you update this to app/^? I realized that there is a page called "login" on the front-end, but thats also the name of an endpoint, so that would cause a conflict

Reply:
@amalik2 it doesn't work like that, `^` matches start of the string. I understand what you are trying to do, but I am not sure how to get it to work. Wouldn't this also require the base path of the react router to be set to `/app`?

Reply:
Ok - got it to work, I added the `basename` in the `BaseRouter`

Reply:
Ah right, I forgot that was regex. If you do just "app/" would it not work? Nothing needs to be configured for react-router, that's based on the path of index.html

Reply:
@amalik2, no - because the react router is basing off the default `/` - thus any navigation on the page will result in a route like `/discover` and not the intended `/app/discover`.

I added the `basename={'/app'}` to the BaseRouter in `index.js` and it is working as intended